.github/workflows/integration-tests-release.yaml

```diff
@@ -0,0 +1,219 @@
+---
+name: Integration Tests and Release
+
+on: 
+  pull_request:
+    paths:
+      - 'pkg/**'
+      - 'cmd/**'
+      - 'charts/**'
+      - 'tests/integration/**'
+      - '.github/workflows/**'
+  push:
+    branches:
+      - 'main'
+      - '**-run-ci'
+      - '*.*'
```

Comment:
Have we decided that tag-branch naming convention?

Reply:
yes, we cut out releases from release branches named in the form: `major.minor`